src/codemagic/__version__.py

```diff
@@ -1,5 +1,5 @@
 __title__ = 'codemagic-cli-tools'
 __description__ = 'CLI tools used in Codemagic builds'
-__version__ = '0.7.5'
+__version__ = '0.0.8'
```

Comment:
New version should be `0.8.0`. Right now you seem to have accidentally reduced version as `0.7.5 > 0.0.8`.